constitution.tex

```diff
@@ -254,7 +254,7 @@ \subsection{Modifying the Constitution}
 
 \asubsection{Active Membership Expectations}
 Active Members are expected to be active participants in CSH as defined in \ref{Expectations of Active Members}.
-Active Voting Members are expected to meet the requirements defined in \ref{Expectations of Voting Members}.
+Active Voting Members are eligible to become Voting Members according to the requirements defined in \ref{Qualifications of Voting Members}.
```

Comment:
As written, this suggests that you are not a voting member until you meet these requirements

Reply:
That's correct. If you don't meet these requirements, you can't vote at six weeks.

Reply:
I got confused about "Voting Members" referring to members who are active and can vote on house issues, but I get it now